### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by redferret**
This PR is the main project setup branch. Introduces ReactJS, Tailwind CSS, with FLUX as a design pattern and React-Router to manage the main page components.

### CODE REVIEW

**Comment by zachjamesgreen** [comment]
everything looks good to go

**Review by zachjamesgreen — APPROVED** [review]

**Review by mkrumholz — APPROVED** [review]
Awesome work team front end! Really excited to see how this plays out. I left some notes and questions, but overall, it looks like you're moving in a cool direction. Even the questions I left aren't critical – didn't find anything that I recognized as concerning, but then, not sure I would yet. Can't wait to see what's next!

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Flux Wallet</title>
+    <meta name="description" content="A simple virtual wallet built with React and Flux.">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <div id="app"></div>
+  </body>
+</html>
```
[the remark this answers is not in the document]
Comment (zachjamesgreen) [reply public/index.html:14]: just added an issue to fix this

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  render() {
+    return (
+      <Router>
+        <Logo />
+        <Link to="/">Home</Link>
+        <Link to="/onboard">Onboard</Link>
+        <Switch>
+          <Route path="/onboard" component={Onboard} />
```
[the remark this answers is not in the document]
Comment (zachjamesgreen) [reply src/components/App.js:60]: The name might change but the component will be the screen that the user will be on when initially rating cocktails

File: src/test/Landing.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+it("renders", () => {
+  act(() => {
+    render(<Landing />, container);
+  });
+  expect(container.textContent).toBe("This is the landing page");
```
[the remark this answers is not in the document]
Comment (zachjamesgreen) [reply src/test/Landing.test.js:24]: Yeah we'll build out more tests as more work is done

**Pull request merged**